sdk/eventhub/azure-eventhubs/tests/livetest/synctests/test_receive.py

```diff
@@ -282,7 +282,9 @@ def test_receive_over_websocket_sync(connstr_senders):
 
 @pytest.mark.liveTest
 def test_receive_run_time_metric(connstr_senders):
-    pytest.skip("Disabled for uamqp 1.2.2. Will enable after uamqp 1.2.3 is released.")
+    from uamqp import __version__ as uamqp_version
+    if uamqp_version < "1.2.3":
```

Comment:
same to the above version string comparsion

Reply:
Changed